src/guides/v2.3/coding-standards/technical-vision/webapi.md

```diff
@@ -91,6 +91,7 @@ Any new design related to Web API must satisfy the following constraints to keep
 1. Unlimited nesting should be supported during requests for related entities. (For example, get Order => Order Items => Products => Related Products)
 1. Field filtration must be performed with SQL queries. Do not filter on the application layer after you've fetched all possible fields.
 1. Third-party customizations must be done separately for Service Contracts and for GraphQL
+1. Classes marked as APIs SHOULD BE utilized inside GraphQL resolvers when possbile to do so without hurting the performance
```

Comment:
Use bold for emphasis not capitals
```suggestion
1. Classes marked as APIs **should be** utilized inside GraphQL resolvers when possible to do so without affecting performance
```

Reply:
In this instance, and commonly in code standards, we bold things like SHOULD BE because they have clear and important meanings. https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html

Reply:
but the two typos MUST be fixed.

Reply:
Looking at the topic as a whole, don't add any particular emphasis to "should be". There are many instances of "should be" and "must be" in the topic, but not of them are in bold or capitalized.

Reply:
Agreed

Reply:
ok